ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake

```diff
@@ -15,6 +15,11 @@
 #
 # Invoke find_package() for all build and buildtool dependencies.
 #
+# :param REQUIRED_PACKAGES: an optional list of package names that are known
+# required CMake dependencies. For these dependencies, find_package() will be
+# invoked with REQUIRED instead of the default of QUIET.
+# :type REQUIRED_PACKAGES: list of string
```

Comment:
Thanks, I copied it [from here](https://github.com/ament/ament_cmake/blob/7b31d859f3d42b9eb6c84db807d95a5604a6881b/ament_cmake_core/cmake/symlink_install/ament_cmake_symlink_install_append_install_code.cmake#L21) was wondering if it was required like that for parsing. 0e178fee5a415caf30f11ba083ee3c5bb2de914e